conscript/info.rkt

```diff
@@ -2,7 +2,8 @@
 
 (define collection "conscript")
 (define deps
-  '("base"
+  '("congame-web"
```

Comment:
Is this to work around the bug that I introduced by making conscript depend on something it shouldn't, which I'll hopefully fix soon once I update the running study, where Bogdan replied that we'll fix it? If so, then remove this, it needs to be fixed elsewhere, but it is a correctly identified bug.

Reply:
I prefer to leave it in for now since otherwise we get setup errors, but I’ll remove it if you prefer to have the errors as part of impetus to fix the issue.

Reply:
I don't think you can leave it as is since this change introduces a circular dependency. `congame-web` depends on `conscript`, so `conscript` can't depend on `congame-web`. This change resolves the setup warnings, with the packages already installed, but makes it impossible to install either package.

Reply:
Ah, actually, it _is_ possible to install two packages that depend on one-another if you `raco pkg install` them at the same time (i.e. `raco pkg install conscript/ congame-web/`). So, maybe this isn't a big deal.

Reply:
Although for this case, we duplicated some code if I remember correctly, so we should still remove one of the versions and then remove the dependency if applicable.

Reply:
> Ah, actually, it _is_ possible to install two packages that depend on one-another if you `raco pkg install` them at the same time

Yes, that's what I found as well (and how I've written the instructions). I think as long as there isn’t a circular dependency at the `require` level there's no practical issue that I know of. But yes it seems like a minor architectural wart to leave it this way.